tests/integration/test_data/npm_smoketest_lockfile1/bom.json

```diff
@@ -0,0 +1,353 @@
+      "type": "library"
+    },
+    {
+      "name": "https://registry.npmjs.org/accepts/-/accepts-1.3.8.tgz",
+      "version": "1.3.8",
+      "type": "library"
+    },
+    {
+      "name": "https://registry.npmjs.org/array-flatten/-/array-flatten-1.1.1.tgz",
+      "version": "1.1.1",
+      "type": "library"
+    },
+    {
+      "name": "file:bar",
+      "type": "library"
+    },
+    {
+      "name": "git+ssh://git@bitbucket.org/cachi-testing/cachi2-without-deps.git#9e164b97043a2d91bbeb992f6cc68a3d1015086a",
+      "type": "library"
+    },
+    {
+      "name": "git+ssh://git@bitbucket.org/cachi-testing/cachi2-without-deps-second.git#09992d418fc44a2895b7a9ff27c4e32d6f74a982",
+      "type": "library"
+    },
+    {
+      "name": "https://registry.npmjs.org/body-parser/-/body-parser-1.20.1.tgz",
+      "version": "1.20.1",
+      "type": "library"
+    },
+    {
+      "name": "https://registry.npmjs.org/bytes/-/bytes-3.1.2.tgz",
+      "version": "3.1.2",
+      "type": "library"
+    },
+    {
+      "name": "https://github.com/cachito-testing/cachito-npm-without-deps/raw/tarball/cachito-npm-without-deps-1.0.0.tgz",
+      "type": "library"
+    },
+    {
+      "name": "https://registry.npmjs.org/call-bind/-/call-bind-1.0.2.tgz",
+      "version": "1.0.2",
+      "type": "library"
+    },
+    {
+      "name": "https://registry.npmjs.org/classnames/-/classnames-2.3.2.tgz",
+      "version": "2.3.2",
+      "type": "library"
+    },
+    {
+      "name": "https://registry.npmjs.org/colors/-/colors-1.4.0.tgz",
+      "version": "1.4.0",
+      "type": "library"
+    },
+    {
+      "name": "https://registry.npmjs.org/content-disposition/-/content-disposition-0.5.4.tgz",
+      "version": "0.5.4",
+      "type": "library"
+    },
+    {
+      "name": "https://registry.npmjs.org/content-type/-/content-type-1.0.5.tgz",
+      "version": "1.0.5",
+      "type": "library"
+    },
+    {
+      "name": "https://registry.npmjs.org/cookie/-/cookie-0.5.0.tgz",
+      "version": "0.5.0",
+      "type": "library"
+    },
+    {
+      "name": "https://registry.npmjs.org/cookie-signature/-/cookie-signature-1.0.6.tgz",
+      "version": "1.0.6",
+      "type": "library"
+    },
+    {
+      "name": "https://registry.npmjs.org/dateformat/-/dateformat-5.0.3.tgz",
+      "version": "5.0.3",
+      "type": "library"
+    },
+    {
+      "name": "https://registry.npmjs.org/debug/-/debug-2.6.9.tgz",
+      "version": "2.6.9",
+      "type": "library"
+    },
+    {
+      "name": "https://registry.npmjs.org/depd/-/depd-2.0.0.tgz",
+      "version": "2.0.0",
+      "type": "library"
+    },
+    {
+      "name": "https://registry.npmjs.org/destroy/-/destroy-1.2.0.tgz",
+      "version": "1.2.0",
+      "type": "library"
+    },
+    {
+      "name": "https://registry.npmjs.org/ee-first/-/ee-first-1.1.1.tgz",
+      "version": "1.1.1",
+      "type": "library"
+    },
+    {
+      "name": "file:eggs-packages/eggs",
+      "type": "library"
+    },
```

Comment:
The names should be the actual names, not urls / file paths. We might need the urls internally, but not in the SBOM

Reply:
Got it, thank you. I'll fix it.